src/renderer/pages/Onboarding/slides/AvailableOnlineSlide.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { OnboardingSlideProps } from '../helpers';
+
 import { useTranslationContext } from 'renderer/context/LocalContext';
 
-export type AvailableOnlineSlideProps = OnboardingSlideProps;
+export type AvailableOnlineSlideProps = object;
```

Comment:
If the props are of type `object` do we need them? I guess this is for the demo

Reply:
I think will be better to not use props here yeah